website/src/pages/log-streaming/configure-streaming-for-third-party-tools/index.mdx

```diff
@@ -92,6 +90,17 @@ $ $KAFKA_HOME/bin/connect-distributed.sh $KAFKA_HOME/config/connect-distributed.
 
 ### Install Splunk
 
+Typically, those interested in this functionality already have Splunk implemented in their environment and have the technical expertise to configure it with the Kafka messaging services.  
+
+The following instructions describe the steps necessary to create a Splunk instance that is deployed on a workstation using a Docker container.  Additionally, configuring the Splunk system for communication with the IBM Event Streams through Kafka is demonstrated.  
+
+Check the following links to ensure that you have the latest installation pre-requisites satisfied for the platform being used.  
```

Comment:
I think it can be more clear overall when pre-requisites should be addressed at the top of the document, this helps the user to avoid surprises half-way through the activity. I noticed also at the top of the document (and throughout) that there is still language like LogDNA.